sound/soc/sof/mediatek/mt8186/mt8186-loader.c

```diff
@@ -24,7 +24,8 @@ void mt8186_sof_hifixdsp_boot_sequence(struct snd_sof_dev *sdev, u32 boot_addr)
 
 	/* set core boot address */
 	snd_sof_dsp_write(sdev, DSP_SECREG_BAR, ADSP_ALTVEC_C0, boot_addr);
-	snd_sof_dsp_write(sdev, DSP_SECREG_BAR, ADSP_ALTVECSEL, ADSP_ALTVECSEL_C0);
+	snd_sof_dsp_write(sdev, DSP_SECREG_BAR, ADSP_ALTVECSEL,
+			  ADSP_ALTVECSEL_C0 | ADSP_ALTVECSEL_C1);
```

Comment:
You want to remove the Change-Id tag in the commit message, it's only relevant for Chrome trees and need to be pruned for upstream contributions.